client/sites-dashboard-v2/dotcom-style.scss

```diff
@@ -62,6 +62,16 @@
 	}
 }
 
+.wpcom-site .is-group-sites.is-global-sidebar-collapsed {
+	.layout__content {
+		padding: 16px 16px 16px calc(var(--sidebar-width-max));
+	}
```

Comment:
Just a heads up that these styles are seeping, but I'm not sure for what they were originally intended. 

<img width="872" alt="Screenshot 2024-04-30 at 21 39 27" src="https://github.com/Automattic/wp-calypso/assets/43215253/625713ae-75af-45ca-9016-022bf67aaaf5">

Reply:
Thanks for the heads up @Aurorum! We'll take a look and get this fixed up!